Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/StackGrabbingTest.java

```diff
@@ -0,0 +1,52 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+@Autonomous(name = "StackGrabbingTest")
+public class StackGrabbingTest extends BaseAutonomous {
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initialize();
+        grabberCameraPipeline.setRanges(Constants.LOWER_BLACK, Constants.UPPER_BLACK);
+        startCameraWithPipeline(grabberCameraPipeline, grabberCamera, Constants.CAMERA_X, Constants.CAMERA_Y);
+        robotCameraPipeline.setRanges(Constants.LOWER_BLUE, Constants.UPPER_BLUE);
+        startCameraWithPipeline(robotCameraPipeline, robotCamera, Constants.CAMERA_X, Constants.CAMERA_Y);
+        waitForStart();
+        sleep(300);
+        driveGrabber(Constants.GRABBER_INITIALIZE_POSITION);
+        sleep(300);
+        for(int i = 0; i <= 4; i++) {
+            driveSlidesAutonomous((Constants.SLIDE_STACK_FOUR - 150)-(160*i));
+            //sleep(500);
+            centerConeStack(robotCameraPipeline);
+            sleep(300);
+            driveGrabber(Constants.GRABBER_CLOSE_POSITION);
+            sleep(300);
+            driveSlidesAutonomous(Constants.SLIDE_LOW);
+            //sleep(500);
+            driveAutonomous(180, 62);
```

Comment:
I don't remember what the arguments to `driveAutonomous` are, but the second one is giving magic number vibes unless it's just a distance. The first one is clearly an angle and you don't need to make constants for 0, 90, 180, etc type of clear and normal angles.

Reply:
The second one is distance